app/models/chargeback/rates_cache.rb

```diff
@@ -25,7 +25,7 @@ def rates(consumption)
                                                      :parents  => consumption.parents_determining_rate)
 
       if consumption.resource_type == Container.name && rates.empty?
-        rates = [ChargebackRate.find_by(:description => "Default Container Image Rate", :rate_type => "Compute")]
+        rates = [ChargebackRate.find_by(:description => "Default Container Image Rate", :rate_type => "Compute")].compact
```

Comment:
I think you can avoid this `.compact`. if you use ChargebackRate.seed in your spec.

I would prefer that, as your spec is kinda high-level big scenario spec. Let's have everything in when we run such spec.